app/templates/editor/level/tasks-tab.jade

```diff
@@ -0,0 +1,41 @@
+mixin task-row(task)
+  - var taskName = task.get('name');
+  - var isComplete = task.get('complete')
+  - var taskLink = view.defaultTaskLinks[taskName]
+  tr.taskRow(data-task=task)
```

Comment:
Don't attach an entire object to an element. Instead, attach its [cid](http://backbonejs.org/#Model-cid) (every instance of a Model has a unique one, which is useful in this case when it doesn't have an id). Then use the collection's [get method](http://backbonejs.org/#Collection-get) to look it up. You could also just store the name and adjust the code accordingly, but the get method's a bit simpler.

[Example in ClassroomView template](https://github.com/codecombat/codecombat/blob/master/app/templates/courses/classroom-view.jade#L116)
[Example in ClassroomView](https://github.com/codecombat/codecombat/blob/master/app/views/courses/ClassroomView.coffee#L190)